__tests__/api.test.ts

```diff
@@ -4,6 +4,9 @@ import {K8sApi} from '../src/api'
 
 import {mockGitRepo, mockKustomization} from './mocks/mocks'
 
+const testHost =
```

Comment:
this is a bit confusing. Maybe `K8sApi()` should accept a `hostName` or you can use `kc.getCurrentCluster().server` in the tests?

Reply:
Yeah, this is kind of weird. It's not so much about specifying the server for the app itself. That's just pulled from the config or lack there of. The testHost is just for nock. I could try to build it up from the constituent components of localhost:8080 or 0.0.0.0 from any port.